db/migrate/20230123192710_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Following the same approach as for the `Post` table, `Author` table is an alias to `User` table and the link should be to the `AuthorId` in the `Author` table. I think you should have:
- `:likes, :author, foreign_key: { to_table: 'users' }`

Reply:
well resolved! ✔️